tests/examples/wibox/awidget/prompt/hooks.lua

```diff
@@ -1,7 +1,8 @@
 local parent    = ... --DOC_NO_USAGE --DOC_HIDE
 local wibox     = require( "wibox"     ) --DOC_HIDE
 local awful     = { prompt = require("awful.prompt"),--DOC_HIDE
-                    util   = require("awful.util")}--DOC_HIDE
+                    util   = require("awful.util"),
```

Comment:
It's not used but it needs to be declared, otherwise `awful.spawn` is nil.
Lua ignores nil values at the end of the table (`#{1,2,3,nil,nil,nil}` is 3). Without this declaration we end up passing hook with 2 arguments and printing warning.